graph/GuidelinesGraph.md

```diff
@@ -326,6 +316,136 @@ For a complete mapping of error codes to HTTP statuses, see
 
 <a name="api-contract-and-non-backward-compatible-changes"></a>
 
+### Enums
+
+In OData, enum are nominal types representing a subset of the nominal type they use, and are especially useful in cases where certain properties have predefined, limited options.
```

Comment:
Can we rephrase?  There are a lot of nominal types here.
Also I think you meant enums (not enum).